laurent22/joplin
# Mobile: Fixes #9858: Fix deeply-nested subnotebook titles invisible in the folder dropdown

# Summary

Limits the maximum indentation depth in the note list to 2/3rds of its width. This ensures that at least part of a notebook title is visible.

Fixes #9858.

# Screenshot

![screenshot: Shows level 2-11 subnotebooks, levels 7-11 all have the same indentation](https://github.com/laurent22/joplin/assets/46334387/d7b90cea-25d1-46e7-be5e-ac73977a8220)

# Testing

1. Create a large number of nested subnotebooks
2. Open a note and change its parent notebook using a dropdown
3. Rotate the phone and repeat steps 1-2. Verify that the maximum indentation is larger when the screen width is larger.

This has been tested successfully on Android 7.
